control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -2389,6 +2390,7 @@ func (r *reconciler) reconcileAzureCloudNodeManager(ctx context.Context, image s
 							VolumeSource: corev1.VolumeSource{
 								HostPath: &corev1.HostPathVolumeSource{
 									Path: "/etc/kubernetes",
+									Type: &emptyHostPathType,
```

Comment:
I think you could get rid of L2323 and just replace this line with `to.Ptr(corev1.HostPathUnset)`

Reply:
@bryan-cox I think you mistyped `ptr.To(corev1.HostPathUnset)` :)